src/main/java/minicraft/core/io/ControllerHandler.java

```diff
@@ -0,0 +1,30 @@
+package minicraft.core.io;
+
+import com.studiohartman.jamepad.*;
+import minicraft.util.Logging;
+import org.tinylog.Logger;
+
+public class ControllerHandler extends ControllerManager {
+	private ControllerManager controllerManager = new ControllerManager();
+	private ControllerIndex controllerIndex;
+
+	public ControllerHandler() {
+		controllerManager.initSDLGamepad();
+		controllerIndex = controllerManager.getControllerIndex(0);
+		controllerManager.update();
+		try {
+			Logger.debug("Controller Detected: " + controllerManager.getControllerIndex(0).getName());
+		} catch (ControllerUnpluggedException e) {
+			Logging.CONTROLLER.debug("No Controllers Detected, moving on.");
```

Comment:
I think these two could also Logging.CONTROLLER

Reply:
Fixed, I missed this.